include/mapnik/proj_transform.hpp

```diff
@@ -50,8 +49,8 @@ class MAPNIK_DECL proj_transform : private util::noncopyable
     bool backward (double *x, double *y , double *z, int point_count, int offset = 1) const;
     bool forward (geometry::point<double> & p) const;
     bool backward (geometry::point<double> & p) const;
-    unsigned int forward (geometry::line_string<double> & ls) const;
-    unsigned int backward (geometry::line_string<double> & ls) const;
+    unsigned int forward (std::vector<geometry::point<double>> & ls) const;
```

Comment:
because of geometry.hpp implementation ^ e.g `std::vector<geometry::point<double>` is a base type.
( there was a ticket re: this but seems disappeared)